File: Harvest/Utilities/HeightFormatter.swift
import Foundation

enum HeightFormatter {
    static func feetAndInches(from centimeters: Int) -> (feet: Int, inches: Int) {
        let totalInches = Int((Double(centimeters) / 2.54).rounded())
        return (totalInches / 12, totalInches % 12)
    }

    static func string(from centimeters: Int) -> String {
        let converted = feetAndInches(from: centimeters)
        return "\(converted.feet)'\(converted.inches)\""
    }
}

File: android/app/src/main/java/com/harvestglass/harvest/util/HeightFormatter.kt
package com.harvestglass.harvest.util

import kotlin.math.roundToInt

/** Port of Harvest/Utilities/HeightFormatter.swift. */
object HeightFormatter {

    fun feetAndInches(centimeters: Int): Pair<Int, Int> {
        val totalInches = (centimeters / 2.54).roundToInt()
        return totalInches / 12 to totalInches % 12
    }

    fun string(centimeters: Int): String {
        val (feet, inches) = feetAndInches(centimeters)
        return "$feet'$inches\""
    }
}
